app/components/avo/views/resource_index_component.html.erb

```diff
@@ -86,6 +86,11 @@
           <%= render Avo::PaginatorComponent.new pagy: @pagy, turbo_frame: turbo_frame || 'none', index_params: @index_params, resource: @resource, parent_record: parent_record, discreet_pagination: field&.discreet_pagination %>
         </div>
       <% end %>
+      <% if view_type.to_sym == :calendar %>
+        <div id="records_panel mt-6">
```

Comment:
```suggestion
        <div id="records_panel" class="mt-6">
```

Reply:
@yshmarov WDYT?

Reply:
Actually the whole div around the component serves 0 purpose => I deleted it :)